tasks/yesno.py

```diff
@@ -0,0 +1,301 @@
+
+from __future__ import print_function
+from __future__ import division
+
+import pickle
+
+from keras.callbacks import EarlyStopping, ModelCheckpoint
+import keras.preprocessing.sequence as prep
+import numpy as np
+
+from pysts.kerasts import graph_input_anssel
+import pysts.loader as loader
+import pysts.nlp as nlp
+from pysts.vocab import Vocabulary
+from keras.layers.core import Activation, Dropout, TimeDistributedDense, Dense
+from keras.regularizers import l2
+from keras.layers.embeddings import Embedding
+from keras.models import Graph
+from pysts.clasrel_layers import Reshape_, WeightedMean
+import pysts.nlp as nlp
+
+from . import AbstractTask
+
+
+class Container:
+    """Container for merging questions together."""
```

Comment:
Isn't it about merging atexts (etexts) for the same question, rather than merging questions?